_projects/gitmate_plugins.md

```diff
@@ -1,8 +1,10 @@
 ---
 name: "Implement GitMate Plugins for coala"
-desc: "GitMate.io helps by automating tasks on GitHub and GitLab. There are new plugins that are specific to coala that should be implemented. "
+desc: "GitMate.io helps by automating tasks on GitHub and GitLab. There are new plugins that are specific to coala that
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.